app/scss/styles.scss

```diff
@@ -68,6 +73,7 @@
 
       .search-heading {
         font-size: 1.5rem;
+        margin: 0;
```

Comment:
I don't think this does anything

Reply:
it actually does. this one had a margin of 10px. This is added to reset that.